clinica/iotools/converters/ixi_to_bids/ixi_to_bids_utils.py

```diff
@@ -0,0 +1,428 @@
+        return "occupation"
+    if column == "QUALIFICATION_ID":
+        return "qualification"
+    if column == "IXI_ID":
+        return "source_id"
+    if column == "DOB":
+        return "date of birth"
+    if column == "STUDY_DATE":
+        return "acq_time"
+    else:
+        return column.lower()
+
+
+def _get_sex_mapping() -> pd.DataFrame:
+    return pd.DataFrame({"SEX": ["male", "female"]}, index=[1, 2])["SEX"]
+
+
+def _get_ethnic_mapping(clinical_data_path: Path) -> pd.DataFrame:
+    # todo : how to test this efficiently ? (feels more like non regression than unit testing maybe)
+    return pd.read_excel(
+        clinical_data_path / "IXI.xls", sheet_name="Ethnicity"
+    ).set_index("ID")["ETHNIC"]
+
+
+def _get_marital_mapping(clinical_data_path: Path) -> pd.DataFrame:
+    return pd.read_excel(
+        clinical_data_path / "IXI.xls", sheet_name="Marital Status"
+    ).set_index("ID")["MARITAL"]
+
+
+def _get_occupation_mapping(clinical_data_path: Path) -> pd.DataFrame:
+    return pd.read_excel(
+        clinical_data_path / "IXI.xls", sheet_name="Occupation"
+    ).set_index("ID")["OCCUPATION"]
+
+
+def _get_qualification_mapping(clinical_data_path: Path) -> pd.DataFrame:
+    return pd.read_excel(
+        clinical_data_path / "IXI.xls", sheet_name="Qualification"
+    ).set_index("ID")["QUALIFICATION"]
+
+
+def read_clinical_data(clinical_data_path: Path) -> pd.DataFrame:
+    """
+    Reads and formats IXI clinical data.
+
+    Parameters
+    ----------
+    clinical_data_path : Path to the directory where the clinical data .xls is stored.
+
+    Returns
+    -------
+    A dataframe containing the clinical data, with some modifications: padded study id and added session id.
+
+    """
+    clinical_data = pd.read_excel(clinical_data_path / "IXI.xls")
+    clinical_data.drop("DATE_AVAILABLE", axis=1, inplace=True)
+    clinical_data["SEX_ID (1=m, 2=f)"] = clinical_data["SEX_ID (1=m, 2=f)"].map(
+        _get_sex_mapping()
+    )
+    clinical_data["ETHNIC_ID"] = clinical_data["ETHNIC_ID"].map(
+        _get_ethnic_mapping(clinical_data_path)
+    )
+    clinical_data["MARITAL_ID"] = clinical_data["MARITAL_ID"].map(
+        _get_marital_mapping(clinical_data_path)
+    )
+    clinical_data["OCCUPATION_ID"] = clinical_data["OCCUPATION_ID"].map(
+        _get_occupation_mapping(clinical_data_path)
+    )
+    clinical_data["QUALIFICATION_ID"] = clinical_data["QUALIFICATION_ID"].map(
+        _get_qualification_mapping(clinical_data_path)
+    )
+    clinical_data["IXI_ID"] = clinical_data.IXI_ID.apply(
+        lambda x: "IXI" + "0" * (3 - len(str(x))) + str(x)
+    )
+    clinical_data.rename(lambda x: _rename_clinical_data(x), axis=1, inplace=True)
+    clinical_data.fillna("n/a", inplace=True)
+    clinical_data["session_id"] = "ses-M000"
+    return clinical_data
+
+
+def _rename_modalities(input_mod: str) -> str:
+    if input_mod == "T1":
+        return "T1w"
+    elif input_mod == "T2":
```

Comment:
Minor suggestion: you can use "if" instead of "elif" and you don't need the last "else".